code/modules/admin/sql_notes.dm

```diff
@@ -117,8 +121,8 @@
 			var/err = query_update_note.ErrorMsg()
 			log_game("SQL ERROR editing note. Error : \[[err]\]\n")
 			return
-		log_admin("[key_name(usr)] has edited [target_ckey]'s note made by [adminckey] from \"[old_note]\" to \"[new_note]\"")
-		message_admins("[key_name_admin(usr)] has edited [target_ckey]'s note made by [adminckey] from \"[old_note]\" to \"[new_note]\"")
+		log_admin("[usr ? key_name(usr) : adminckey] has edited [target_ckey]'s note made by [adminckey] from \"[old_note]\" to \"[new_note]\"")
```

Comment:
Here it's incorrect. adminckey can be just a previous admin since it's retrieved from the DB itself. Here you lack an identifier as to what system could have done it so just a string or such will do.